### ISSUE TITLE
Noise errors in pinniped-supervisor logs

### ISSUE THREAD

**Issue by doddatpivotal** [vmware-tanzu/pinniped#480]
**What happened?**

I created a management cluster using TKG 1.3 release RC2.  After the management cluster was created, I started receiving these logs entries in the pinniped-supervisor.  These log entries continue un-ending forever.

pinniped-supervisor/pinniped-supervisor-9b7df7976-f6hzv[pinniped-supervisor]: I0307 23:18:48.157097 1 log.go:181] http: TLS handshake error from 100.96.2.1:53914: EOF
pinniped-supervisor/pinniped-supervisor-9b7df7976-f6hzv[pinniped-supervisor]: I0307 23:18:57.593025 1 log.go:181] http: TLS handshake error from 100.96.2.1:30722: EOF
pinniped-supervisor/pinniped-supervisor-9b7df7976-g5kzh[pinniped-supervisor]: I0307 23:18:58.157253 1 log.go:181] http: TLS handshake error from 100.96.2.1:42869: EOF

**What did you expect to happen?**

I expected no error messages with a clean deployment.

**What is the simplest way to reproduce this behavior?**

**In what environment did you see this bug?**
Tanzu Kubernets Grid 1.3 RC2

**What else is there to know about this bug?**

**Comment by enj** [comment]
This is normally caused by something trying to connect via `http` instead of `https`.  Maybe some kind of health check?

**Comment by doddatpivotal** [comment]
Would this do it?

    Liveness:     http-get http://:8080/healthz delay=2s timeout=15s period=10s #success=1 #failure=5
    Readiness:    http-get http://:8080/healthz delay=2s timeout=3s period=10s #success=1 #failure=3

**Comment by enj** [comment]
> Would this do it?
> 
> ```
> Liveness:     http-get http://:8080/healthz delay=2s timeout=15s period=10s #success=1 #failure=5
> Readiness:    http-get http://:8080/healthz delay=2s timeout=3s period=10s #success=1 #failure=3
> ```

Yeah I think so, I opened #482 to try and fix it.  You could try editing the deployment in your cluster in a similar way to see if it helps with the noise.

**Comment by enj** [comment]
Looking at the code, 8080 is the HTTP port, so that should not be causing this issue.
